test/decode_cms.py

```diff
@@ -1,9 +1,11 @@
 #!/usr/bin/env python3
 
 import inspect
+from datetime import datetime
+
 from pyasn1.codec.der import decoder
 from pyasn1.type import univ, char
-from pyasn1_modules import rfc2315, rfc5280, rfc5480, rfc5652
+from pyasn1_modules import rfc2315, rfc5280, rfc5480, rfc5652, rfc5751
```

Comment:
A short line of motivation in the commit message (e.g., which information we intend to obtain here for which future checks) would help readers.